app/javascript/shared/components/Branding.vue

```diff
@@ -47,7 +54,6 @@ export default {
 };
 </script>
 
-<!-- Add "scoped" attribute to limit CSS to this component only -->
```

Comment:
https://github.com/chatwoot/chatwoot/pull/2685/files#diff-339c53cb115f488df2b3099dc110a67d36e3ab5a9b7611fc114a73dc385bf881R50-R52


You might want to disable this method as well if service pack is loaded.